server/json/user1-institutions.json

```diff
@@ -6,47 +6,53 @@
       "status": {
         "code": 0,
         "message": ""
-      }
+      },
+      "currentSubmission": 0
```

Comment:
Good point on the timestamp. Couldn't we also use the `id` from the `submissions.json`? (I could be missing what the `currentSubmission` is doing for us.)